include/net/cloud.h

```diff
@@ -260,6 +275,44 @@ static inline int cloud_input(const struct cloud_backend *const backend)
 	return backend->api->input(backend);
 }
 
+/**@brief Add Cloud endpoint subscriptions.
+ *
+ * @param backend Pointer to cloud backend structure.
+ * @param endpoint Pointer to a list of endpoint subscriptions.
```

Comment:
The description does not match the actual argument list.

Reply:
Fixed.